configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -133,6 +133,28 @@ BEGIN
 END
 #
 /*
+This function accepts patient/person id and returns that person's age in years
+*/
+#
+DROP FUNCTION IF EXISTS current_age_in_years;
+#
+CREATE FUNCTION current_age_in_years(
+    _person_id int)
+
+    RETURNS int
+    DETERMINISTIC
+
+BEGIN
+    DECLARE currentAge DOUBLE;
```

Comment:
You changed the return value to an int correctly, but you also need this to be an int as well, no?